code/game/turfs/closed/walls.dm

```diff
@@ -9,7 +9,7 @@
 	explosion_block = 1
 
 	thermal_conductivity = WALL_HEAT_TRANSFER_COEFFICIENT
-	heat_capacity = 312500 //a little over 5 cm thick , 312500 for 1 m by 2.5 m by 0.25 m plasteel wall
+	heat_capacity = 50000 //also indicates the temperature at wich the wall will melt (currently only able to melt with H/E pipes)
```

Comment:
Why 50k? is it just what felt right? or what
~~The comment's cute, can we keep it?~~

Reply:
50k feels the right amount of heat that a normal iron wall should be able to handle (tho iron melts at way lower temperature). Having it at 300k allowed the walls to soak too much heat (like fusion amounts of heat) before even reaching 300k K

Reply:
you know the wall will heat up over time right? so it's not like the first tick of heat injection needs to melt it.
have ya tested it in game? might be a good way to get a feel for things

Reply:
I know that, I tested it without changing anything and by using a canister at 1e6 K with 1500 moles of plasma the wall only reached 1000 K.
With the change a normal wall is able to handle the heat well enough and will take time before melting down

Reply:
since it's totally arbitrary anyway: how about 312500 (the original value) for rwalls and 62500 (the original value divided by 5) for normal walls?

Reply:
sure is close enough